src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.cs

```diff
@@ -95,7 +95,7 @@ ImmutableArray<CompletionItem> GetItemsFromCacheResult(GetCacheResult cacheResul
             {
                 var compilation = await referencedProject.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
                 var assembly = SymbolFinder.FindSimilarSymbols(compilation.Assembly, currentCompilation, cancellationToken).SingleOrDefault();
-                var metadataReference = currentCompilation.GetMetadataReference(assembly);
+                var metadataReference = assembly != null ? currentCompilation.GetMetadataReference(assembly) : null;
```

Comment:
`GetMetadataReference` is annotated to not allow null, but both implementation doesn't check for null. Maybe there can be a change.

Reply:
@genlu?

Reply:
This looks correct to me, but I don't think we should change `GetMetadataReference` to handle null.